### DISCUSSION TITLE
Update Github Action settings to ignore markdown files

### DISCUSSION THREAD

**Discussion by zion830** [euphony-io/euphony#192] [General]
I want to modify the GitHub action setting files in Euphony.

### AS-IS
* Github Action runs when only markdown files are modified. But markdown files don't affect  the build process, so it is unnecessary

### TO-BE
* By using `paths-ignore`, commits modifying markdown files doesn't run Github Action

**Comment by designe** [comment]
Resolved through https://github.com/euphony-io/euphony/pull/194,
